msteams-platform/toolkit/TeamsFx-CLI.md

```diff
@@ -1,23 +1,462 @@
 ---
 title: TeamsFx Command Line Interface
 author: MuyangAmigo
-description: In this module, learn TeamsFx Command Line Interface, TeamsFx library, supported commands and its scenarios
+description: In this module, learn Teamsapp Command Line Interface, TeamsFx library, supported commands and its scenarios
 ms.author: zhany
 ms.localizationpriority: medium
 ms.topic: overview
 ms.date: 11/29/2021
 zone_pivot_groups: toolkit-cli
 ---
 
-# TeamsFx command line interface
+# Teams Toolkit command line interface
 
-TeamsFx CLI is a text-based command line interface that accelerates Teams application development. It aims to provide keyboard centric experience while building Teams applications.
+> [!IMPORTANT]
+>
+> We recommend that you use Teams Toolkit CLI v3 for building your Teams app. TeamsFx CLI v1 and TeamsFx CLI v2 will soon be deprecated.
+
+Teams Toolkit command line interface (Teams Toolkit CLI) is a text-based command line interface that accelerates Teams application development. It aims to provide keyboard centric experience while building Teams applications.
 
 For more information, see:
 
 * [Source code](https://github.com/OfficeDev/TeamsFx/tree/dev/packages/cli)
 * [Package (NPM)](https://www.npmjs.com/package/@microsoft/teamsfx-cli)
 
+::: zone pivot="version-three"
+
+Teams Toolkit CLI encapsulates common functionality and integration patterns, such as simplified access to Microsoft Identity. You can build apps for Microsoft Teams with zero configuration.
+
+Following is a list of main Teams Toolkit CLI features:
+
```

Comment:
We can update the list of main Teams Toolkit CLI features to:
* Create new Teams application from templates and samples
* Preview your Teams application by uploading the custom app to Teams, Outlook and the Microsoft 365 app.
* Provision cloud resources and deploy the application to Azure
* Validate, package and publish application.
* Manage multiple environments, Microsoft entra-apps and Teams app registrations.
* Collaborate with other developers.

Reply:
Updated.